### PR TITLE
Reference for colors

### PR DESCRIPTION
**Pull request by tiffwong**
I thought I would create a way to easier reference and visualize the colors used in the theme, 
[http://localhost:6006/?path=/story/color-palette--default](http://localhost:6006/?path=/story/color-palette--default)

### CODE REVIEW

File: packages/manager/src/components/ColorPalette/ColorPalette.tsx [hunk 1]
```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import { makeStyles, Theme } from 'src/components/core/styles';
+import Typography from 'src/components/core/Typography';
+import Grid from 'src/components/Grid';
+// eslint-disable-next-line no-restricted-imports
```
Comment (tiffwong) [inline packages/manager/src/components/ColorPalette/ColorPalette.tsx:5]: Is there a better way to handle this?

File: packages/manager/src/components/ColorPalette/ColorPalette.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+}
+
+export type CombinedProps = Props;
+
+export const ColorPalette: React.FC<CombinedProps> = props => {
+  const classes = useStyles();
+  const theme = useTheme();
+
+  const primaryColors = [
+    { color: theme.palette.primary.main, alias: 'theme.palette.primary.main' },
```
Comment (tiffwong) [inline packages/manager/src/components/ColorPalette/ColorPalette.tsx:44]: I couldn't figure out how to render the color as just a string without it automatically converting to a hex

File: packages/manager/src/components/ColorPalette/ColorPalette.tsx (hunk 1 above)
Comment (Jskobos) [reply packages/manager/src/components/ColorPalette/ColorPalette.tsx:5]: `import { useTheme } from 'src/components/core/styles` will work I think

File: packages/manager/src/components/ColorPalette/ColorPalette.tsx (hunk 2 above)
Comment (Jskobos) [reply packages/manager/src/components/ColorPalette/ColorPalette.tsx:44]: Not quite sure what this means in practice, but I like having the hex values there.

File: packages/manager/src/components/ColorPalette/ColorPalette.tsx (hunk 1 above)
Comment (tiffwong) [reply packages/manager/src/components/ColorPalette/ColorPalette.tsx:5]: If I do that, it won't recognize the color/palette/bg properties for some reason 🤔

File: packages/manager/src/components/ColorPalette/ColorPalette.tsx (hunk 2 above)
Comment (tiffwong) [reply packages/manager/src/components/ColorPalette/ColorPalette.tsx:44]: I was wondering if there's a better way to display theme.palette.primary.main and etc. without having to write it a second time wrapped in single quotes

**Pull request merged**